src/encoded/tests/test_batch_download.py

```diff
@@ -146,6 +179,22 @@ def test_files_prop_param_list(test_input, expected):
     assert test_input == expected
 
 
+@pytest.mark.parametrize('test_url,expected', [
```

Comment:
same, this test is very fragile in terms of the test set, which changes slowly but it does change.

Reply:
Same logic as before- Do you think it is "pythonic" to have fragile test?